### ISSUE TITLE
Form LineEdit: addind placeholders & inputmask

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#278]
```
Add the possibility to define an input mask and a placeholder in lineedits in 
forms
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 21 Jan 2013 at 8:30

**Comment by GoogleCodeExporter** [comment]
```
Issue 265 has been merged into this issue.
```

Original comment by `eric.mae...@gmail.com` on 21 Jan 2013 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 5c1c0c101f8c.
```

Original comment by `eric.mae...@gmail.com` on 22 Jan 2013 at 9:51
- Changed state: **Fixed**
